src/ctap/mod.rs

```diff
@@ -26,10 +26,10 @@ pub mod hid;
 pub mod key_material;
 mod large_blobs;
 pub mod main_hid;
-mod pin_protocol;
+pub(crate) mod pin_protocol;
 pub mod response;
 pub mod status_code;
-mod storage;
+pub(crate) mod storage;
```

Comment:
I assume this is for tests? I suspect that there is a design issue if all these crates have to suddenly become public.

Reply:
Yes, any better ideas of this? test_helpers isn't under ctap but it needs to access many private things inside.

Reply:
We should probably achieve what we want through the public API. While this is more work the first few times we do this, it should become easier the more helpers we have. WDYT?